templates/bake/Plugin/config/routes.php.twig

```diff
@@ -18,12 +18,13 @@ declare(strict_types=1);
 
 use Cake\Routing\Route\DashedRoute;
 use Cake\Routing\RouteBuilder;
-use Cake\Routing\Router;
 
-Router::plugin(
-    '{{ plugin }}',
-    ['path' => '/{{ routePath }}'],
-    function (RouteBuilder $routes) {
-        $routes->fallbacks(DashedRoute::class);
-    }
-);
+$routes
+    ->setRouteClass(DashedRoute::class)
```

Comment:
This will change the default class for routes added outside of the plugin as well. You should move this inside the plugin scope to preserve the previous behavior.

Reply:
My bad, fixed.